### PR TITLE
Fix cert naming to sig

### PR DESCRIPTION
**Pull request by Gabib32**
use the correct terminology; this is checking the validity
of the manifest.MoM.sig file, not the cert. Add check that
the manifest.MoM and manifest.MoM.sig files exist prior to
running the check.

Signed-off-by: gabrielle beyer <gabrielle.n.beyer@intel.com>

### CODE REVIEW

File: cmd/certs.go [hunk 1]
```diff
@@ -11,42 +12,43 @@ import (
...
+	sigCmd.Flags().StringVar(&sigFlags.certpath, "certpath", "/usr/share/clear/update-ca/Swupd_Root.pem", "fully qualified path to ca-cert")
 }
 
-var certsCmd = &cobra.Command{
-	Use:   "certs",
-	Short: "",
-	Long:  ``,
-	Run:   runCheckCerts,
+var sigCmd = &cobra.Command{
+	Use:   "sig",
```
Comment (matthewrsj) [inline cmd/certs.go]: Can we call it "signature" instead? I don't really like using an abbreviation like "sig"

File: cmd/certs.go [hunk 2]
```diff
@@ -61,6 +63,12 @@ func verifyMoM(r *diva.Results, baseCache, version, cert string) error {
 	mom := filepath.Join(cache, "Manifest.MoM")
 	sig := filepath.Join(cache, "Manifest.MoM.sig")
 
+	_, momExists := os.Stat(mom)
+	_, sigExists := os.Stat(sig)
+	if os.IsNotExist(momExists) || os.IsNotExist(sigExists) {
```
Comment (matthewrsj) [inline cmd/certs.go]: +1 for this check, but it should be a different commit from the `certs`->`sig` change.

File: cmd/certs.go [hunk 3]
```diff
@@ -61,6 +63,12 @@ func verifyMoM(r *diva.Results, baseCache, version, cert string) error {
...
+		return fmt.Errorf("manifest.MoM or manifest.MoM.sig does not exist at %s for version %s. Try running diva [download|fetch] update -v [version]", cache, version)
```
Comment (matthewrsj) [inline cmd/certs.go]: can you please manually wrap this line

File: cmd/certs.go [hunk 4]
```diff
@@ -62,6 +63,13 @@ func verifyMoM(r *diva.Results, baseCache, version, cert string) error {
 	mom := filepath.Join(cache, "Manifest.MoM")
 	sig := filepath.Join(cache, "Manifest.MoM.sig")
 
+	_, momExists := os.Stat(mom)
+	_, sigExists := os.Stat(sig)
+	if os.IsNotExist(momExists) || os.IsNotExist(sigExists) {
+		return fmt.Errorf(`manifest.MoM or manifest.MoM.sig does not exist at %s for version %s.
+Try running diva [download|fetch] update -v [version]`, cache, version)
```
Comment (matthewrsj) [inline cmd/certs.go]: This actually adds a newline to the error string itself, which we don't really want... Also capitalize the M in Manifest.

```go
return fmt.Errorf("Manifest.MoM or Manifest.MoM.sig does not exist at %s for version %s. " +
    "Try running diva [download|fetch] update -v [version]", cache, version)
```

You know, this is an error return and error returns don't need to tell the user what to do anyways, that should be handled by the printer if at all.
```go
return fmt.Errorf("Manifest.MoM or Manifest.MoM.sig does not exist at %s for version %s.")
```

Comment (Gabib32) [reply cmd/certs.go]: fiiine

Comment (Gabib32) [reply cmd/certs.go]: grr error strings cannot be capitalized

Comment (matthewrsj) [reply cmd/certs.go]: "the Manifest.MoM or Manifest.MoM.sig file..." maybe?

Comment (matthewrsj) [reply cmd/certs.go]: Actually you can just re-word the error, since you print the filenames anyways.
```go
return fmt.Errorf("%s or %s does not exist at %s for version %s",
    mom, sig, cache, version)
```
